examples/usbus_minimal/Makefile

```diff
@@ -16,18 +16,29 @@ USEMODULE += usbus
 USEMODULE += auto_init_usbus
 
 # USB device vendor and product ID
-DEFAULT_VID = 1209
-DEFAULT_PID = 0001
+export DEFAULT_VID = 1209
+export DEFAULT_PID = 0001
```

Comment:
As this is touched anyway, maybe use http://pid.codes/1209/7D00/ already?

Reply:
True, I will change those

Reply:
Changed PID to `7D00`